exp/api/v1beta1/awsmanagedmachinepool_types.go

```diff
@@ -144,6 +143,9 @@ type AWSManagedMachinePoolSpec struct {
 	// +kubebuilder:default:=onDemand
 	// +optional
 	CapacityType *ManagedMachinePoolCapacityType `json:"capacityType,omitempty"`
+
+	// +optional
+	AWSLaunchTemplate *AWSLaunchTemplate `json:"awsLaunchTemplate,omitempty"`
```

Comment:
It would be good to have a comment here. Maybe worth mentioning that you can't use this if you fields X, Y, Z?

Reply:
Sounds good. Added comment here.